pkg/services/sqlstore/sqlbuilder.go

```diff
@@ -33,61 +41,22 @@ func (sb *SQLBuilder) AddParams(params ...interface{}) {
 }
 
 func (sb *SQLBuilder) WriteDashboardPermissionFilter(user *models.SignedInUser, permission models.PermissionType) {
-	if user.OrgRole == models.ROLE_ADMIN {
-		return
-	}
-
-	okRoles := []interface{}{user.OrgRole}
-
-	if user.OrgRole == models.ROLE_EDITOR {
-		okRoles = append(okRoles, models.ROLE_VIEWER)
+	var (
+		sql    string
+		params []interface{}
+	)
+	if !ac.IsDisabled(sb.cfg) {
+		sql, params = permissions.NewAccessControlDashboardPermissionFilter(user, permission, "").Where()
+	} else {
+		sql, params = permissions.DashboardPermissionFilter{
```

Comment:
question: I haven't properly compared them, but I'm guessing that `permissions.DashboardPermissionFilter` creates the same query as [the one that was removed here](https://github.com/grafana/grafana/pull/53379/files#diff-e52cea9531bf5ebd28e5ea1ba700c513383b80429980d1e11a39369ea10cac3fL48-L86)?

Reply:
Yes it was copy paste of the query